keps/sig-cluster-lifecycle/image-builder/20190612-image-builder.md

```diff
@@ -0,0 +1,126 @@
+---
+kep-number: 37
+title: Kubernetes Image Builder
+status: provisional
+authors:
+  - "@timothysc"
+  - "@moshloop"
+owning-sig: sig-cluster-lifecycle
+reviewers:
+  - "@justinsb"
+  - "@luxas" 
+  - "@astrieanna"
+approvers:
+  - "@justinsb"
+  - "@timothysc"
+  - "@luxas"
+editor:
```

Comment:
This is an optional field and there can, according to the template, be only one editor